src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIMarsManager.cs

```diff
@@ -18,7 +17,7 @@ internal class SNIMarsManager
         /// <summary>
         /// Constructor
         /// </summary>
-        public SNIMarsManager()
+        private SNIMarsManager()
```

Comment:
Unrelated to this change, I'm curious... why is this a singleton, rather than just having the various instance methods and state instead be static?